{"content": {"suites": {"0": {"status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 2, "total_error": 0}, "tests": {"0": {"status": "FAIL", "message": "E   pytest_bdd.exceptions.StepDefinitionNotFoundError: Step definition is not found: Then \"X-Cache-Key header is present in response\". Line 37 in scenario \"when cache is invalidated\" in the feature \"/Users/richa.jha/Desktop/NHS_Proxy/proxy-validated-relationships-service/tests/test_api/features/PDSCache/1_pds_cache.feature\"\n", "test_name": "test_when_cache_is_invalidated[9730676399-empty_value]", "rerun": "0"}, "1": {"status": "FAIL", "message": "E   pytest_bdd.exceptions.StepDefinitionNotFoundError: Step definition is not found: Then \"X-Cache-Key header is present in response\". Line 37 in scenario \"when cache is invalidated\" in the feature \"/Users/richa.jha/Desktop/NHS_Proxy/proxy-validated-relationships-service/tests/test_api/features/PDSCache/1_pds_cache.feature\"\n", "test_name": "test_when_cache_is_invalidated[9730676399-RelatedPerson]", "rerun": "0"}}, "suite_name": "test_api/test_pds_cache.py"}}}, "date": "July 19, 2024", "start_time": 1721400394.351093, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "0", "fail": "2", "skip": "0", "error": "0", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "2"}